### PR TITLE
devinfo.handlebars: Add comserv.duckduckgo.com in the list of dev ser…

### PR DESCRIPTION
**Pull request by jagtalon**
…vers.

To @jdorweiler or @jbarrett

### CODE REVIEW

**Comment by jagtalon** [comment]
screen shot 2015-08-20 at 12 48 17 pm

File: src/templates/devinfo.handlebars [hunk 1]
```diff
@@ -48,6 +48,7 @@
                 {{#not_eq test_machine 'ddh8'}}<option value="8">ddh8</option>{{/not_eq}}
                 {{#not_eq test_machine 'ddh9'}}<option value="9">ddh9</option>{{/not_eq}}
                 {{#not_eq test_machine 'ddh10'}}<option value="10">ddh10</option>{{/not_eq}}
+                {{#not_eq test_machine 'comserv'}}<option value="11">comserv</option>{{/not_eq}}
```
Comment (jdorweiler) [inline src/templates/devinfo.handlebars]: can you change to "alpha"

**Comment by jagtalon** [comment]
@jdorweiler Changed.

**Comment by jdorweiler** [comment]
:+1:

**Pull request merged**